cognite_toolkit/_repo_files/AzureDevOps/.devops/deploy-pipeline.yml

```diff
@@ -25,5 +25,7 @@ jobs:
       - checkout: self
       - script: cdf build
         displayName: 'Build the modules'
+      - script: cdf auth verify --no-prompt
+        displayName: 'Verify required permissions'
```

Comment:
The disadvantage of this is if you do not have GROUP create capabilities. Then, it will fail, even though you can deploy other resources.

Reply:
I assume the initial toolkit principal group setup will ensure it has permission to create or edit groups. However, we can configure the GitHub Action or Azure DevOps to proceed even if this step fails, without interrupting the entire workflow. This step is intended to ensure that any additional required permissions are added to the principal group before deployment.